src/components/main-menu/MainMenu.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import cases from "jest-in-case";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import MainMenu from "./MainMenu";
+
+test("should display title", () => {
+  render(<MainMenu />);
+
+  expect(screen.getByText(/atom/i)).toBeInTheDocument();
```

Comment:
is `toBeInTheDocument` better or worse than `toBeVisible` for this sort of thing?

Reply:
I had to think about this one, and go digging through what the matchers do :stuck_out_tongue: 

I think it may make more sense to use `toBeVisible`, as we want users to actually be able to read it :thinking: